### ISSUE TITLE
index.html

### ISSUE THREAD

**Issue by riku-1346** [riku-1346/practice3#1]
<!DOCTYPE html>
<html lang="ja">
  <head>
    <meta charset="utf-8">
    <title>travel</title>
    <meta name="description" content="テキストテキストテキストテキストテキストテキストテキストテキス">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="img/favicon.ico">
    <link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.css">
    <link rel="stylesheet" href="style.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.js"></script>
    <script src="script.js"></script>
  </head>

  <body>
    <header id="header">
      <h1 class="site-title">
        <a href="index.html">Travel</a>
      </h1>

      <nav id="navi">
        <ul class="nav-menu">
          <li><a href="#pickup">PICK UP</a></li>
          <li><a href="#scenery">Scenery</a></li>
          <li><a href="#contact">CONTACT</a></li>
        </ul>

        <ul class="nav-sns">
          <li><a href="https://twitter.com/" target="_blank">Twitter</a></li>
          <li><a href="https://www.facebook.com/" target="_blank">Facebook</a></li>
          <li><a href="https://www.instagram.com/" target="_blank">Instagram</a></li>
        </ul>
      </nav>

      <div class="toggle_btn">
        <span></span>
        <span></span>
        <span></span>
      </div>

      <div id="mask"></div>
    </header>

      <main id="main">
        <div id="video">
          <video id="bg-video" src="images/travel.mp4" loop autoplay muted playsinline></video>
        </div>

        <section id="pickup">
          <h2 class="sec-title">PICK UP</h2>
          <ul class="slick-area">
            <li><img src="images/cherry-blossoms2.jpg" alt="ピックアップ1　桜"></li>
            <li><img src="images/beach2.jpg" alt="ピックアップ2 海"></li>
            <li><img src="images/autumn2.jpg" alt="ピックアップ3　紅葉"></li>
            <li><img src="images/winter2.jpg" alt="ピックアップ4　雪"></li>
            <li><img src="images/mount2.jpg" alt="ピックアップ5　山"></li>
            <li><img src="images/river2.jpg" alt="ピックアップ6　川"></li>
            <li><img src="images/lake2.jpg" alt="ピックアップ7　湖"></li>
            <li><img src="images/shrine2.jpg" alt="ピックアップ8　寺"></li>
            <li><img src="images/starry-sky2.jpg" alt="ピックアップ9　月"></li>
          </ul>
        </section>

        <section id="scenery">
          <h2 class="sec-title">SCENERY</h2>
          <div class="grid">
            <div class="item">
              <img class="fadein" src="images/cherry-blossoms.jpg" alt="">
              <div class="item-content">
                <p class="item-cat">category</p>
                <p class="item-text">テキストテキストテキストテキスト</p>
                <p class="item-date">XXXX.XX.XX</p>
              </div>
            </div>

            <div class="item">
              <img class="fadein" src="images/beach.jpg" alt="">
              <div class="item-content">
                <p class="item-cat">category</p>
                <p class="item-text">テキストテキストテキストテキスト</p>
                <p class="item-date">XXXX.XX.XX</p>
              </div>
            </div>

            <div class="item">
              <img class="fadein" src="images/autumn.jpg" alt="">
              <div class="item-content">
                <p class="item-cat">category</p>
                <p class="item-text">テキストテキストテキストテキスト</p>
                <p class="item-date">XXXX.XX.XX</p>
              </div>
            </div>

              <div class="item">
                <img class="fadein" src="images/winter.jpg" alt="">
                <div class="item-content">
                  <p class="item-cat">category</p>
                  <p class="item-text">テキストテキストテキストテキスト</p>
                  <p class="item-date">XXXX.XX.XX</p>
                </div>
              </div>

              <div class="item">
                <img class="fadein" src="images/mount.jpg" alt="">
                <div class="item-content">
                  <p class="item-cat">category</p>
                  <p class="item-text">テキストテキストテキストテキスト</p>
                  <p class="item-date">XXXX.XX.XX</p>
                </div>
              </div>

              <div class="item">
                <img class="fadein" src="images/river.jpg" alt="">
                <div class="item-content">
                  <p class="item-cat">category</p>
                  <p class="item-text">テキストテキストテキストテキスト</p>
                  <p class="item-date">XXXX.XX.XX</p>
                </div>
              </div>

              <div class="item">
                <img class="fadein" src="images/lake.jpg" alt="">
                <div class="item-content">
                  <p class="item-cat">category</p>
                  <p class="item-text">テキストテキストテキストテキスト</p>
                  <p class="item-date">XXXX.XX.XX</p>
                </div>
              </div>

              <div class="item">
                <img class="fadein" src="images/shrine.jpg" alt="">
                <div class="item-content">
                  <p class="item-cat">category</p>
                  <p class="item-text">テキストテキストテキストテキスト</p>
                  <p class="item-date">XXXX.XX.XX</p>
                </div>
              </div>
             
              <div class="item">
                <img class="fadein" src="images/starry-sky.jpg" alt="">
                <div class="item-content">
                  <p class="item-cat">category</p>
                  <p class="item-text">テキストテキストテキストテキスト</p>
                  <p class="item-date">XXXX.XX.XX</p>
                </div>
              </div>
          </div>
        </section> 

        <section id="contact">
          <h2 class="sec-title">CONTACT</h2>
          <div class="content">
            <div class="contact-info">
              <p>テキストテキストテキスト</p>
              <p>
                テキストテキストテキストテキストテキストテキストテキストテキストテキスト
                テキストテキストテキストテキストテキストテキストテキストテキストテキスト
                テキストテキストテキストテキストテキストテキストテキストテキストテキスト
              </p>
              <p>
                テキストテキストテキストテキストテキストテキストテキストテキストテキスト
                テキストテキストテキストテキストテキストテキストテキストテキストテキスト
                テキストテキストテキストテキストテキストテキストテキストテキストテキスト
              </p>
            </div>

           <div class="contact-form">
             <form action="#">
               <dl>
                 <dt><label for="name">Name:</label></dt>
                 <dd><input type="text" id="name" name="your-name"></dd>
                 <dt><label for="email">Mail:</label></dt>
                 <dd><input type="text" id="email" name="your-email"></dd>
                 <dt><label for="message">Message</label></dt>
                 <dd><textarea name="your-message" id="message"></textarea></dd>
               </dl>
               <div class="button"><input type="submit" value="SEND"></div>
             </form>
           </div>
          </div>
        </section>
      </main>

      <footer id="footer">
        <p>&copy; travel</p>
      </footer>

  </body>
</html>

**Comment by riku-1346** [comment]
@charset "UTF-8";

html {
  font-size: 100%;
}

body {
  background-color: #FFF;
  color: #121212;
  font-size: 0.9rem;
  line-height: 1.7;
}

a {
  color: #121212;
  text-decoration: none;
}

img {
  max-width: 100%;
}

li {
  list-style: none;
}

.site-title {
  line-height: 1px;
  padding: 40px;
  font-size: 80px;
  font-family: serif;
}

.site-title a {
  display: block;
}

.sec-title {
  font-size: 50px;
  font-family: serif;
  margin-bottom: 30px;
  text-align: center;
}

/*---フェード表示させる要素に使用するためのクラス---*/

.fadein {
  opacity : 0;
  transform: translateY(20px);
  transition: all 1s;
}

/*---ヘッダー---*/

#header {
  width: 100%;
  display: flex;
  justify-content: space-between;
  padding: 20px 40px;
  position: fixed;
  z-index: 10;
  background-color: #dddddd;
}

#navi {
  display: block;
  position: fixed;
  top: 0;
  left: -300px;
  bottom: 0;
  width: 300px;
  color: #fff;
  padding: 60px 25px;
  background-color: #121212;
  overflow-x: hidden;
  overflow-y: auto;
  /*-webkit-overflow-scrolling: touch;*/
  transition: all .5s;
  z-index: 20;
  opacity: 0;
}

#navi a {
  color: #fff;
}

.open #navi {
  left: 0;
  opacity: 1;
}

#navi ul {
  margin: 0;
  padding: 0;
}

#navi ul.nav-menu {
  margin-bottom: 60px;
  border-top: solid 1px #fff;
}

#navi ul.nav-menu li {
  position: relative;
  margin: 0;
  padding: 20px 0;
  border-bottom: solid 1px #fff;
}

#navi ul.nav-sns li {
  padding: 5px 0;
}

.toggle_btn {
  display: block;
  position: fixed;
  top: 25px;
  right: 45px;
  width: 30px;
  height: 30px;
  transition: all .5s;
  cursor: pointer;
  z-index: 20;
}

.toggle_btn span {
  display: block;
  position: absolute;
  left: 0;
  width: 30px;
  height: 2px;
  background-color: #333;
  border-radius: 4px;
  transition: all .5s;
}

.toggle_btn span:nth-child(1) {
  top: 4px;
}

.toggle_btn span:nth-child(2) {
  top: 14px;
}

.toggle_btn span:nth-child(3) {
  bottom: 4px;
}

.open .toggle_btn span {
  background-color: #fff;
}

.open .toggle_btn span:nth-child(1) {
  -webkit-transform: translateY(10px) rotate(-315deg);
  transform: translateY(10px) rotate(-315deg);
}

.open .toggle_btn span:nth-child(2) {
  opacity: 0;
}

.open .toggle_btn span:nth-child(3) {
  -webkit-transform: translateY(-10px) rotate(315deg);
  transform: translateY(-10px) rotate(315deg);
}

#mask {
  display: none;
  transition: all .5s;
}

.open #mask {
  display: block;
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: #000;
  opacity: .8;
  z-index: 10;
  cursor: pointer;
}

/*---Main---*/

#main {
  padding-top: 80px;
}

/*---Video---*/

#bg-video {
  width: 100%;
}

/*---Pickup---*/

#pickup {
  width: 100%;
  padding: 100px 0 50px 0;
}

#pickup .slick-area {
  font-size: 0;
}

#pickup .slick-area li {
  padding: 0 30px;
}

/*---Scenery---*/

#scenery {
  max-width: 1240px;
  padding: 100px 16px 50px 16px;
  margin: -50px auto 10px auto;
}

#scenery .grid {
  display: grid;
  gap: 26px;
  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
}

#scenery .grid .item {
  transition: all 0.5s ease;
  box-shadow: 0 0 8px 4px #AAAAAA;
}

#scenery .grid .img {
  vertical-align: top;
}




#scenery .grid .item-content {
  padding: 30px;
}
#scenery .grid .item-cat {
  font-size: 0.75rem;
  margin-bottom: 20px;
}
#scenery .grid .item-text {
  font-weight: bold;
  margin-bottom: 20px;
}
#scenery .grid .item-date {
  font-size: 0.75rem;
  text-align: right;
}
/*-------------------------------------------
Contact
-------------------------------------------*/
#contact {
  color: #fff;
  background-color: #121212;
  padding: 50px 0;
}
#contact .sec-title {
  color: #fff;
}
#contact .content {
  max-width: 1240px;
  padding: 0 16px;
  margin: 0 auto;
  display: flex;
  justify-content: space-between;
}
#contact .contact-info p {
  margin-bottom: 30px;
}
#contact .contact-info,
#contact .contact-form {
  width: 45%;
}
#contact .contact-form input,
#contact .contact-form textarea {
  width: 100%;
  background-color: #fff;
  padding: 10px;
  margin-bottom: 20px;
}
#contact .contact-form .button input {
  width: 200px;
  color: #fff;
  background-color: #121212;
  border: solid 1px #fff;
  padding: 12px 0;
  margin-bottom: 0;
}
#contact .contact-form .button input:hover {
  color: #202020;
  background-color: #fff;
}

/*-------------------------------------------
フッター
-------------------------------------------*/
#footer {
  color: #fff;
  background-color: #121212;
  text-align: center;
  padding: 10px;
  font-size: 0.75rem;
}

/*---スマホ---*/

@media screen and (max-width: 600px) {
 .sec-title {
    font-size: 1.5rem;
    margin-bottom: 20px;
  }

/*---ヘッダー---*/
  #header {
    padding: 20px 16px;
  }
  .toggle_btn {
    right: 20px;
  }

/*---Video---*/
  
  #bg-video {
    width: 100%;
    height: 400px;
    object-fit: cover;
  }


  /*-------------------------------------------
  Pickup
  -------------------------------------------*/
  #pickup {
    padding: 80px 0;
  }
  #pickup .slick-area li {
    padding: 0 10px;
  }

  /*-------------------------------------------
  Scenery
  -------------------------------------------*/
  #scenery {
    padding: 80px 16px;
  }
  #scenery .grid .item-content {
    padding: 16px;
  }

  /*-------------------------------------------
  Contact
  -------------------------------------------*/
  #contact {
    padding: 80px 0;
    height: 1200px;
  }
  #contact .content {
    flex-direction: column;
  }
  #contact .contact-info,
  #contact .contact-form {
    width: 100%;
  }
  #contact .contact-form .button input {
    width: 100%;
  }
}

**Comment by riku-1346** [comment]
![apple](https://user-images.githubusercontent.com/93707011/159537093-5af16712-b0a1-4c22-9c66-e7e55e114354.jpg)
![autumn](https://user-images.githubusercontent.com/93707011/159537160-912e80b0-b993-476e-9bab-64e8ed828f6b.jpg)
![autumn2](https://user-images.githubusercontent.com/93707011/159537241-2277c4a9-6b54-44b9-8523-4f7bc1894b10.jpg)
![banana](https://user-images.githubusercontent.com/93707011/159537268-0a856886-f558-4629-8b0b-69d2a5bcc6db.jpg)
![beach2](https://user-images.githubusercontent.com/93707011/159537299-9c2e87e5-e0de-4022-a786-76b2426ed86d.jpg)
![cherry-blossoms](https://user-images.githubusercontent.com/93707011/159537421-2e21f63e-33ca-47ca-b144-e72c17e51e3a.jpg)
![cherry-blossoms2](https://user-images.githubusercontent.com/93707011/159537439-3e89c259-13f0-4f5b-a963-71ed9aa33714.jpg)
![lake](https://user-images.githubusercontent.com/93707011/159537454-656775bd-dc00-495b-acab-e12809eda776.jpg)
![lake2](https://user-images.githubusercontent.com/93707011/159537482-a6dc8faa-2d9e-497a-8897-f227fd6f46fa.jpg)
![mount2](https://user-images.githubusercontent.com/93707011/159537495-217f77bd-b46c-4a33-ab46-7e6154e63cba.jpg)
![river](https://user-images.githubusercontent.com/93707011/159537508-c13abe1a-2b65-4f14-a63e-ad964af0e3cd.jpg)
![river2](https://user-images.githubusercontent.com/93707011/159537520-b75f338f-ea79-44f4-aefe-b56b9826af4f.jpg)
![road](https://user-images.githubusercontent.com/93707011/159537529-8bcf3c36-16fe-42be-b5a1-c09318512376.jpg)
![shrine](https://user-images.githubusercontent.com/93707011/159537535-d73ef2c9-0fae-4872-afca-8aa16b8eb104.jpg)
![shrine2](https://user-images.githubusercontent.com/93707011/159537543-06c0bf69-688c-49cb-ad12-18c8fdc2f7a6.jpg)
![starry-sky](https://user-images.githubusercontent.com/93707011/159537554-a1af6bf4-bd02-4345-9d7f-ee752fea4931.jpg)
![starry-sky2](https://user-images.githubusercontent.com/93707011/159537565-61bc7b65-458e-414b-bf4a-f4c66119b7d3.jpg)
![travel logo](https://user-images.githubusercontent.com/93707011/159537577-e6e4089d-42ea-462a-8471-3724e10b1efb.jpg)
![winter](https://user-images.githubusercontent.com/93707011/159537608-194014fa-8abd-4b3e-9bde-d1cb06408fe0.jpg)
![winter2](https://user-images.githubusercontent.com/93707011/159537625-693ba9a6-72d4-4340-86df-da91ab315608.jpg)
